bims/migrations/0088_auto_20181207_1310.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-12-07 13:10
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def rename_reference_category_consulting_report(apps, schema_editor):
+
+    my_model = apps.get_model('bims', 'biologicalcollectionrecord')
+
+    for mm in my_model.objects.all():
```

Comment:
rather just do filter
```suggestion
    for mm in my_model.objects.filter(refernce_category='Consulting report'):
```